python/src/aac/persistence/load.py

```diff
@@ -0,0 +1,57 @@
+"""Module for loading data from the state file and making it available to the rest of the AaC package."""
```

Comment:
I like the idea of having a module that's dedicated to persistence for other persistence concerns like user configuration, but these functions seem wholly concerned with the LanguageContext. I've been thinking about giving the LanguageContext a directory module structure into which we could drop these functions into a module like `aac.lang.language_context.persistent_state`.